pkg/cmd/cli/cmd/set/imagelookup.go

```diff
@@ -250,7 +311,24 @@ func (o *ImageLookupOptions) printImageLookup(infos []*resource.Info) error {
 	defer w.Flush()
 	fmt.Fprintf(w, "NAME\tLOCAL\n")
 	for _, info := range infos {
-		fmt.Fprintf(w, "%s\t%t\n", info.Name, info.Object.(*imageapi.ImageStream).Spec.LookupPolicy.Local)
+		switch t := info.Object.(type) {
+		case *imageapi.ImageStream:
+			fmt.Fprintf(w, "%s\t%t\n", info.Name, t.Spec.LookupPolicy.Local)
+		default:
+			accessor, ok := ometa.GetAnnotationAccessor(info.Object)
+			if !ok {
+				fmt.Fprintf(w, "%s/%s\tUNKNOWN\n", info.Mapping.Resource, info.Name)
```

Comment:
This isn't an error message, this is indicating that the provided object doesn't support the annotation.  We don't display errors in list views usually, we display something simple.  Effectively, this is "unsupported".